uxarray/plot/dataarray_plot.py

```diff
@@ -214,65 +213,29 @@ def _point_raster(
 
     if uxda._face_centered():
         # data mapped to face centroid coordinates
-        lon = uxda.uxgrid.face_lon.values
-        lat = uxda.uxgrid.face_lat.values
+        lon, lat = uxda.uxgrid.face_lon.values, uxda.uxgrid.face_lat.values
     elif uxda._node_centered():
         # data mapped to face corner coordinates
-        lon = uxda.uxgrid.node_lon.values
-        lat = uxda.uxgrid.node_lat.values
+        lon, lat = uxda.uxgrid.node_lon.values, uxda.uxgrid.node_lat.values
     elif uxda._edge_centered():
         # data mapped to face corner coordinates
-        lon = uxda.uxgrid.edge_lon.values
-        lat = uxda.uxgrid.edge_lat.values
+        lon, lat = uxda.uxgrid.edge_lon.values, uxda.uxgrid.edge_lat.values
     else:
         raise ValueError(
             f"The Dimension of Data Variable {uxda.name} is not Node or Face centered."
         )
 
-    # determine whether we need to recompute points, typically when a new projection is selected
-    recompute = True
-    if uxda._face_centered() == "center":
-        if (
-            uxda.uxgrid._centroid_points_df_proj[0] is not None
-            and uxda.uxgrid._centroid_points_df_proj[1] == projection
-        ):
-            recompute = False
-            points_df = uxda.uxgrid._centroid_points_df_proj[0]
-
-    else:
-        if (
-            uxda.uxgrid._corner_points_df_proj[0] is not None
-            and uxda.uxgrid._corner_points_df_proj[1] == projection
-        ):
-            recompute = False
-            points_df = uxda.uxgrid._corner_points_df_proj[0]
-
-    if recompute:
-        # need to recompute points and/or projection
-        if projection is not None:
-            lon, lat, _ = projection.transform_points(ccrs.PlateCarree(), lon, lat).T
-
-        point_dict = {"lon": lon, "lat": lat, "var": uxda.values}
-
-        # Construct Dask DataFrame
-        point_ddf = dd.from_dict(data=point_dict, npartitions=npartitions)
+    if projection is not None:
```

Comment:
This change seems to be beyond the scope; please clarify

Reply:
There's no changes to the functionality, other than this PR does not cache the results due to the issues with Dask.

#739 will address this in the future, after some research into best ways to handle storing these dask objects.

Reply:
I am so sorry; I missed your [yesterday's comment](https://github.com/UXARRAY/uxarray/pull/731#issuecomment-2013472875), which clarifies everything, before giving this review.

Reply:
No worries!